### DISCUSSION TITLE
Uncaught ReferenceError: Livewire is not defined

### DISCUSSION THREAD

**Discussion by NigelDorning** [livewire/livewire#3429] [1. Help]
Hi,

I'm just deploying a Laravel/Livewire application but running into this issue: _Uncaught ReferenceError: Livewire is not defined_. It's giving me a 404 error on the route to the livewire js file but if I go to that route I'm shown the javascript so it looks like it should be working.

The Laravel application is in a folder above the root directory like it should be.

Thanks a lot

**Comment by NigelDorning** [comment]
Okay publishing Livewire assets seems to have fixed that. I'm now getting a 500 error on the components though.

**Comment by joshhanley** [comment]
@NigelDorning if you are now getting a 500 error, can you make sure you are on the latest version of Livewire 2.5.5? As there was a bug in production in a previous patch release.

If that doesn't work, can you share some more details? Is loading any component throwing the 500 or just some? Did you do anything else to trigger the error?
